test/token/ERC20/extensions/ERC4626.t.sol

```diff
@@ -15,22 +15,4 @@ contract ERC4626StdTest is ERC4626Test {
         _vaultMayBeEmpty = false;
```

Comment:
shouldn't be `_vaultMayBeEmpty = true;` now? See https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3792#issuecomment-1302682873

Reply:
Thanks for the heads up, fixed in #4068

Reply:
awesome